docs/UserGuide.md

```diff
@@ -316,16 +316,10 @@ Notes about search keywords:
 
 * You must provide at least one keyword to search for.<br>
   e.g. entering just `find n/` alone is not a valid command as the keyword is empty.
-
-* Tags must only have one keyword.<br>
-  e.g. `find t/zoom math` is invalid. To search by multiple tags, you can do `find t/zoom t/math`.
   
 * The search is case-insensitive.<br>
   e.g. keyword `hans` will match `Hans`.
 
-* A keyword can match a word partially.<br>
-  e.g. keyword `math` will match `mathematics`.
```

Comment:
I don't think removing solves it as it's not addressed in table? I'll think about how better to represent this.